docs/PythonAPIOverview.md

```diff
@@ -214,3 +214,17 @@ import onnx.utils
 model = onnx.load('path/to/the/model.onnx')
 polished_model = onnx.utils.polish_model(model)
 ```
+
+### Updating Model's Inputs Outputs with Variable Length
+Function `update_inputs_outputs_dims` updates the dimension of the inputs and outputs of the model,
```

Comment:
I think we should explain more about how we handle dynamic/variable dimension in ONNX. You may link the corresponding doc as well.